test/javascripts/turbolinks_test.coffee

```diff
@@ -1,210 +1,957 @@
+      200,
+      {'Content-Type':'text/html'},
+      """
+        <!doctype html>
+        <html>
+          <head>
+            <script src='#{ASSET_FIXTURES['b.js']}'
+              data-turbolinks-track="true"
+              data-turbolinks-track-script-as="b"
+              type="text/javascript"></script>
+            <script src='#{ASSET_FIXTURES['c.js']}'
+              data-turbolinks-track="true"
+              data-turbolinks-track-script-as="c"
+              type="text/javascript"></script>
+            <script src='#{ASSET_FIXTURES['a.js']}'
+              data-turbolinks-track="true"
+              data-turbolinks-track-script-as="a"
+              type="text/javascript"></script>
+            <title>Hi there!</title>
+          </head>
+          <body>
+            <div id="turbo-area" refresh="turbo-area"></div>
+          </body>
+        </html>
+      """
+    ],
+
+    threeScriptsInHeadCAB: [
+      200,
+      {'Content-Type':'text/html'},
+      """
+        <!doctype html>
+        <html>
+          <head>
+            <script src='#{ASSET_FIXTURES['c.js']}'
+              data-turbolinks-track="true"
+              data-turbolinks-track-script-as="c"
+              type="text/javascript"></script>
+            <script src='#{ASSET_FIXTURES['a.js']}'
+              data-turbolinks-track="true"
+              data-turbolinks-track-script-as="a"
+              type="text/javascript"></script>
+            <script src='#{ASSET_FIXTURES['b.js']}'
+              data-turbolinks-track="true"
+              data-turbolinks-track-script-as="b"
+              type="text/javascript"></script>
+            <title>Hi there!</title>
+          </head>
+          <body>
+            <div id="turbo-area" refresh="turbo-area"></div>
+          </body>
+        </html>
+      """
+    ],
+
+    threeScriptsInHeadCBA: [
+      200,
+      {'Content-Type':'text/html'},
+      """
+        <!doctype html>
+        <html>
+          <head>
+            <script src='#{ASSET_FIXTURES['c.js']}'
+              data-turbolinks-track="true"
+              data-turbolinks-track-script-as="c"
+              type="text/javascript"></script>
+            <script src='#{ASSET_FIXTURES['b.js']}'
+              data-turbolinks-track="true"
+              data-turbolinks-track-script-as="b"
+              type="text/javascript"></script>
+            <script src='#{ASSET_FIXTURES['a.js']}'
+              data-turbolinks-track="true"
+              data-turbolinks-track-script-as="a"
+              type="text/javascript"></script>
+            <title>Hi there!</title>
+          </head>
+          <body>
+            <div id="turbo-area" refresh="turbo-area"></div>
+          </body>
+        </html>
+      """
+    ],
+
+    singleLinkInHead: [
+      200,
+      {'Content-Type':'text/html'},
+      """
+        <!doctype html>
+        <html>
+          <head>
+            <link href='#{ASSET_FIXTURES['foo.css']}' data-turbolinks-track="true"></link>
```

Comment:
Can `foo.css`/`bar.css` become `a.css`/`b.css`?  It'd cut down on the fixture count.